src/styles/dropdown.ts

```diff
@@ -17,43 +17,89 @@ export const dropdownCss = css`
     position: absolute;
     z-index: 1000;
 
+    // scroll
+    max-height: 80vh;
+    overflow-y: auto;
+
```

Comment:
sorry this is my fault - i actually shipped this change in a [recent PR](https://github.com/code4recovery/tsml-ui/commit/61df4c0d6900715e43f981f308c5f24c3917fe20#diff-58322c92a35f7d291218784c65e62e0bac9a65d3674f7923293a44ae69e824e4R13-R16), do you mind reverting this change?